Dynamodb (#996)
micronaut-projects/micronaut-guides: src/docs/common/snippets/common-run-dynamodb-with-docker.adoc

> For local develoment, we can https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html[setup DynamoDB locally].

Reviewer: ```suggestion
For local development, we can https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html[set up DynamoDB locally].
```

> An ease option is to run it via Docker.

Reviewer: ```suggestion
An easy option is to run it via Docker.
```

> After installing Docker, execute the following command to run a DynamoDB Local container:

Reviewer: ```suggestion
After installing Docker, execute the following command to run a DynamoDB local container:
```
